auth/src/main/java/com/firebase/ui/auth/util/Builder.java

```diff
@@ -0,0 +1,5 @@
+package com.firebase.ui.auth.util;
+
+public interface Builder<T> {
```

Comment:
This interface is not necessary, builders having a `build()` method is a known Java convention and I don't see any methods that are dependent on the generic `Builder` interface in this PR ( correct me if I am wrong)